ert_gui/gert_main.py

```diff
@@ -232,16 +232,15 @@ def main(argv):
     ert = EnKFMain(res_config, strict=strict, verbose=verbose)
     configureErtNotifier(ert, config_file)
 
-    window = GertMainWindow()
-    window.setWidget(SimulationPanel())
-    window.setWindowTitle('ERT - {}'.format(config_file))
+    window = GertMainWindow(config_file)
+    window.setWidget(SimulationPanel(config_file))
     plugin_handler = PluginHandler(ert, ert.getWorkflowList().getPluginJobs(), window)
 
     help_tool = HelpTool("ERT", window)
 
     window.addDock("Configuration Summary", SummaryPanel(), area=Qt.BottomDockWidgetArea)
-    window.addTool(IdeTool(os.path.basename(config_file), help_tool))
-    window.addTool(PlotTool())
+    window.addTool(IdeTool(config_file, os.path.basename(config_file), help_tool))
```

Comment:
You could have reduced the number of args at this point, extracting ```os.path.basename(config_file)``` inside constructor of IdeTool just to make it a bit cleaner.